src/blueprints/tv2/factories/tv2-atem-video-switcher-timeline-factory.ts

```diff
@@ -25,8 +26,8 @@ export class Tv2AtemVideoSwitcherTimelineFactory implements Tv2VideoSwitcherTime
             cutSource: downstreamKeyer.Key
           },
           properties: {
-            clip: downstreamKeyer.Clip * 10, // input is percents (0-100), atem uses 1-000
-            gain: downstreamKeyer.Gain * 10, // input is percents (0-100), atem uses 1-000
+            clip: this.convertPercentageToAtemPercentageValue(downstreamKeyer.Clip), // input is percents (0-100), atem uses 1-000
```

Comment:
The point of making this method was so you could remove the comment....

Reply:
Or atleast move the comment to the new method, so we only have to have it once...

Reply:
Moved, and reworded, to a TsDoc Return on the method.